packages/dd-trace/test/appsec/index.fastify.plugin.spec.js

```diff
@@ -29,13 +30,10 @@ withVersions('fastify', 'fastify', '>=2', version => {
         reply.send('DONE')
       })
 
-      getPort().then((port) => {
-        app.listen({ port }, () => {
-          axios = Axios.create({ baseURL: `http://localhost:${port}` })
-          done()
-        })
-        server = app.server
-      })
+      const port = await getPort()
```

Comment:
we need to stop using getPort, just let the OS choose a port for you, by passing no port argument, and getting the assigned one with app.server.address().port or soemthing like that

Reply:
check https://github.com/DataDog/dd-trace-js/pull/6060 for examples